src/test/java/com/google/api/generator/engine/writer/JavaWriterVisitorTest.java

```diff
@@ -179,6 +180,53 @@ public void writeVariableExpr_scopedStaticFinalDecl() {
     assertThat(writerVisitor.write()).isEqualTo("public static final boolean x");
   }
 
+  @Test
+  public void writeTernaryExpr() {
+    IdentifierNode identifier = IdentifierNode.builder().setName("x").build();
```

Comment:
May need to merge PR #64 since the API changed.

Reply:
sure!

Reply:
seems #64 gets merged but the changes are not in this PR.

Reply:
Should be fine for merging now.

Reply:
sounds great!